systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthPlainST.java

```diff
@@ -193,10 +195,18 @@ void testProducerConsumerMirrorMaker() throws IOException, InterruptedException,
                 .endSpec()
                 .done();
 
-        consumer = oauthKafkaClient.receiveMessages(TOPIC_NAME, NAMESPACE, targetKafkaCluster, MESSAGE_COUNT,
-                CONSUMER_GROUP_NAME + "-" + rng.nextInt(Integer.MAX_VALUE));
-
-        assertThat(consumer.get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS), is(MESSAGE_COUNT));
+        TestUtils.waitFor("Waiting for Mirror Maker will copy messages from " + CLUSTER_NAME + " to " + targetKafkaCluster,
+            Duration.ofSeconds(30).toMillis(), Constants.TIMEOUT_FOR_MIRROR_MAKER_COPY_MESSAGES_BETWEEN_BROKERS,
+            () -> {
+                try {
+                    Future<Integer> oauthConsumer = oauthKafkaClient.receiveMessages(TOPIC_NAME, NAMESPACE, targetKafkaCluster,
+                            MESSAGE_COUNT, CONSUMER_GROUP_NAME + "-" + rng.nextInt(Integer.MAX_VALUE));
+                    return MESSAGE_COUNT == oauthConsumer.get(Constants.GLOBAL_CLIENTS_TIMEOUT, TimeUnit.MILLISECONDS);
+                } catch (IOException | InterruptedException | ExecutionException | TimeoutException e) {
+                    LOGGER.info("Consumer doesn't received messages {}", e.getMessage());
```

Comment:
out of curiosity, why an exception is logged as INFO and not as ERROR printing the stack trace? Is it the pattern that you are using everywhere?

Reply:
Good point, It will make more sense if I change it to error :)